api/src/main/java/org/openmrs/module/commonreports/reports/MSPPAntenatalReportManager.java

```diff
@@ -348,7 +348,7 @@ public ReportDefinition constructReportDefinition() {
 		fifthVisit.setOperator(SetComparator.IN);
 		fifthVisit.setQuestion(inizService.getConceptFromKey("report.MSPP.antenatal.visitNumber"));
 		fifthVisit.addValue(inizService.getConceptFromKey("report.MSPP.antenatal.fivePlus"));
-		antenatalGestation.addColumn(col5, createCohortComposition(fifthVisit), null);
+		antenatalGestation.addColumn(col5, createCohortComposition(fifthVisit), parameterMappings);
 		
 		// Total Visit column
 		CodedObsCohortDefinition totalVisit = new CodedObsCohortDefinition();
```

Comment:
@rbuisson here I can see that no sql clause has been added not sure why it was done this way. @kdaud could you verify if the right most `Total` column gives the correct values as per time range?